Probe/src/probe/rpc.go

```diff
@@ -44,12 +46,20 @@ var (
 // Retrieve metadata from string manually from flattened format instead of using JSON unmarshalling
 // because data is deeply nested, and unmarshalling JSON would require several nested structs or type assertions
 func getProbeData(raw string) (*controller.MetadataConfig, error) {
-	items := strings.Split(raw, "\n")
+	items := strings.Split(raw, "commonInstanceMetadata.items")
 	var probeData []string
 	for i, item := range items {
 		// Search for item "probeData" key, manipulate the associated value at the next index
 		if strings.Contains(item, "probeData") {
+			// data will come in the form of: 'value: "DATA"'
 			probeData = strings.SplitN(items[i+1], ": ", 2)
+			// Remove trailing newline character from cert for unquoting
+			probeData[1] = probeData[1][:len(probeData[1]) - 1]
```

Comment:
strings.Trim() would be better for this.

Reply:
addressed in 024773b

Reply:
addressed in 024773b